samples/corda/corda-simple-application/clients/src/main/resources/config/credentials/Corda_Network2/access-control.json

```diff
@@ -5,5 +5,17 @@
     "principalType": "certificate",
     "resource": "localhost:10006#com.cordaSimpleApplication.flow.GetStateByKey:*",
     "read": true
+  },
+  {
+    "principal": "-----BEGIN CERTIFICATE-----\nMIIByzCCAWegAwIBAgIQaj/tzScOOq5mA6XOZ6xlXTAUBggqhkjOPQQDAgYIKoZI\nzj0DAQcwLzELMAkGA1UEBhMCR0IxDzANBgNVBAcMBkxvbmRvbjEPMA0GA1UECgwG\nUGFydHlBMB4XDTIxMTAyNjAwMDAwMFoXDTI3MDUyMDAwMDAwMFowLzELMAkGA1UE\nBhMCR0IxDzANBgNVBAcMBkxvbmRvbjEPMA0GA1UECgwGUGFydHlBMCowBQYDK2Vw\nAyEAVxGzRmahlqiYY1Xae7a469UYL0ST44myWj1lnAF/yuWjgYkwgYYwHQYDVR0O\nBBYEFA1+fHn5Aqe61gxRVrYYveplXCWqMA8GA1UdEwEB/wQFMAMBAf8wCwYDVR0P\nBAQDAgKEMBMGA1UdJQQMMAoGCCsGAQUFBwMCMB8GA1UdIwQYMBaAFFYWFwBR5VMh\ni6Wr+/ldDSWUODgbMBEGCisGAQQBg4piAQEEAwIBBjAUBggqhkjOPQQDAgYIKoZI\nzj0DAQcDSAAwRQIhAPFlXeyOPM/NR05cSNwbX2TFomwV04sN7eZFpCkzVz5HAiAj\nNGmNhIbeiktfehc3tdUGkf7BR6RovI4rLqMlDo8MYg\u003d\u003d\n-----END CERTIFICATE-----",
+    "principalType": "certificate",
+    "resource": "localhost:10006#net.corda.samples.tokenizedhouse.flows.GetAssetPledgeStatusByPledgeId:*",
+    "read": true
+  },
+  {
+    "principal": "-----BEGIN CERTIFICATE-----\nMIIByzCCAWegAwIBAgIQaj/tzScOOq5mA6XOZ6xlXTAUBggqhkjOPQQDAgYIKoZI\nzj0DAQcwLzELMAkGA1UEBhMCR0IxDzANBgNVBAcMBkxvbmRvbjEPMA0GA1UECgwG\nUGFydHlBMB4XDTIxMTAyNjAwMDAwMFoXDTI3MDUyMDAwMDAwMFowLzELMAkGA1UE\nBhMCR0IxDzANBgNVBAcMBkxvbmRvbjEPMA0GA1UECgwGUGFydHlBMCowBQYDK2Vw\nAyEAVxGzRmahlqiYY1Xae7a469UYL0ST44myWj1lnAF/yuWjgYkwgYYwHQYDVR0O\nBBYEFA1+fHn5Aqe61gxRVrYYveplXCWqMA8GA1UdEwEB/wQFMAMBAf8wCwYDVR0P\nBAQDAgKEMBMGA1UdJQQMMAoGCCsGAQUFBwMCMB8GA1UdIwQYMBaAFFYWFwBR5VMh\ni6Wr+/ldDSWUODgbMBEGCisGAQQBg4piAQEEAwIBBjAUBggqhkjOPQQDAgYIKoZI\nzj0DAQcDSAAwRQIhAPFlXeyOPM/NR05cSNwbX2TFomwV04sN7eZFpCkzVz5HAiAj\nNGmNhIbeiktfehc3tdUGkf7BR6RovI4rLqMlDo8MYg\u003d\u003d\n-----END CERTIFICATE-----",
+    "principalType": "certificate",
+    "resource": "localhost:10006#com.cordaSimpleApplication.flow.GetAssetPledgeStatusByPledgeId:*",
+    "read": true
   }]
-}
\ No newline at end of file
+}
```

Comment:
Given the asymmetry of policies specified for the two networks, it looks like transfers are all currently expected to be tested only one way, is it?

Reply:
Tested Fabric-Corda and Corda-Fabric where `network1` is instance of a Fabric network and `Corda_Network` is instance of a Corda network.